ui/component/userEmailVerify/view.jsx

```diff
@@ -46,8 +51,14 @@ class UserEmailVerify extends React.PureComponent<Props> {
 
   handleResendVerificationEmail() {
     const { email, resendVerificationEmail, toast } = this.props;
-    resendVerificationEmail(email);
-    toast(__('New email sent.'));
+    if (!this.state.wait) {
```

Comment:
If `wait` exists, we should show a separate snack bar saying "please wait a bit" or something.

Reply:
done